src/map/skill.c

```diff
@@ -11580,17 +11598,15 @@ static int skill_castend_pos2(struct block_list *src, int x, int y, uint16 skill
 
 		case WL_EARTHSTRAIN:
 			{
-				int i, wave = skill_lv + 4, dir = map->calc_dir(src,x,y);
+				int i, wave = skill_lv + 4;
```

Comment:
Please declare each variable in its own line.
And, maybe, while your at it, move the declaration of `i` to the loop header?

Reply:
Done except for `i`.